tests/integration/targets/cloudtrail/tasks/main.yml

```diff
@@ -1347,9 +1347,9 @@
       state: present
       name: '{{ cloudtrail_name }}'
       kms_key_id: 'alias/{{ kms_alias }}'
-      aws_access_key: "{{ noKms_assumed_role.sts_creds.access_key }}"
-      aws_secret_key: "{{ noKms_assumed_role.sts_creds.secret_key }}"
-      security_token: "{{ noKms_assumed_role.sts_creds.session_token }}"
+      access_key: "{{ noKms_assumed_role.sts_creds.access_key }}"
+      secret_key: "{{ noKms_assumed_role.sts_creds.secret_key }}"
+      session_token: "{{ noKms_assumed_role.sts_creds.session_token }}"
```

Comment:
These creds should not be needed.

Reply:
This is just a rename PR, not sure why they're there so I'd rather handle that separately.